src/component/mxgraph/StyleConfigurator.ts

```diff
@@ -110,9 +113,19 @@ export default class StyleConfigurator {
     style[this.mxConstants.STYLE_SHAPE] = ShapeBpmnElementKind.EVENT_END;
     style[this.mxConstants.STYLE_PERIMETER] = this.mxPerimeter.EllipsePerimeter;
     style[this.mxConstants.STYLE_VERTICAL_ALIGN] = 'top';
+    style[this.mxConstants.STYLE_STROKECOLOR] = '#000';
     this.putCellStyle(ShapeBpmnElementKind.EVENT_END, style);
   }
 
+  private configureEndEventTerminateStyle(): void {
```

Comment:
:question: that was not what I had in mind
I thought we agree on having a single style. When creating the vertex, we pass additional argument to the runtime style which would have then be managed by the Event Shape to decide how to render it.
Or maybe, we will manage it when implementing the rendering of the `Terminate End Event`
Let's talk about it directly

Reply:
I think that would be the work related to #137 ?